k8s/namespaces/idam/idam-user-dashboard/idam-user-dashboard.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-user-dashboard
+  namespace: idam
+spec:
+  releaseName: idam-user-dashboard
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/idam-user-dashboard
+  values:
+    nodejs:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/idam/user-dashboard:prod-* #{"$imagepolicy": "flux-system:idam-user-dashboard"}
+      ingressHost: idam-user-dashboard.aat.platform.hmcts.net
+      environment:
+        STRATEGIC_SERVICE_URL: https://idam-api.aat.platform.hmcts.net
```

Comment:
ingress host + all these environment values should be set in your chart using the global 'environment' variable available

Reply:
Solved in https://github.com/hmcts/cnp-flux-config/pull/13624/commits/a43f752bdac947f685d31c6dc5da009f1f33a3ff - thanks once again @timja!